Add more gradients examples (#33095)
mdn/content: files/en-us/web/css/css_images/using_css_gradients/index.md

{{ EmbedLiveSample('Stacked_gradients', 200, 200) }}

### Blending gradients

> In addition to transparency, stacking multiple semi-transparent gradients and stacking gradients over raster background images, gradients can be used with other CSS effects. In this example, the four {{htmlelement("div")}} elements have the same gradient as a background image and we apply different {{cssxref("background-blend-mode")}} CSS property values to the last three to create blended effects.

Reviewer: ```suggestion
In addition to transparency, stacking multiple semi-transparent gradients and stacking gradients over raster background images, gradients can be used with other CSS effects. In this example, the four {{htmlelement("div")}} elements have the same two fully-opaque gradients as background images. We apply different {{cssxref("background-blend-mode")}} CSS property values to the last three that blend the two background images creating different effects.
```

{{ EmbedLiveSample('Stacked_gradients', 200, 200) }}

### Blending gradients

> In addition to transparency, stacking multiple semi-transparent gradients and stacking gradients over raster background images, gradients can be used with other CSS effects. In this example, we use the {{cssxref("background-blend-mode")}} CSS property to create blended effects.

Reviewer: ```suggestion
In addition to transparency, stacking multiple semi-transparent gradients and stacking gradients over raster background images, gradients can be used with other CSS effects. In this example, the four {{htmlelement("div")}} elements have the same gradient as a background image and we apply different {{cssxref("background-blend-mode")}} CSS property values to the last three to create blended effects.
```

```html hidden

> <div class="normal-gradients"></div>
> <div class="screen-gradients"></div>
> <div class="overlay-gradients"></div>
> <div class="difference-gradients"></div>

Reviewer: ```suggestion
<div class="original"></div>
<div class="screen"></div>
<div class="overlay"></div>
<div class="difference"></div>
```

```

```css hidden
div {
  width: 120px;
  height: 120px;
  float: left;
  margin-right: 10px;
  box-sizing: border-box;
}
```

```css

> .normal-gradients {
>   background: linear-gradient(to top, red, blue), linear-gradient(to right, #5500ff, #00ff55);
> }
> 
> .screen-gradients {
>   background: linear-gradient(to top, red, blue), linear-gradient(to right, #5500ff, #00ff55);
>   background-blend-mode: screen;
> }
> 
> .overlay-gradients {
>   background: linear-gradient(to top, red, blue), linear-gradient(to right, #5500ff, #00ff55);
>   background-blend-mode: overlay;
> }
> 
> .difference-gradients {
>   background: linear-gradient(to top, red, blue), linear-gradient(to right, #5500ff, #00ff55);
>   background-blend-mode: difference;
> }

Reviewer: ```suggestion
div {
  background: linear-gradient(to top, red, blue), linear-gradient(to right, #5500ff, #00ff55);
}

.screen {
  background-blend-mode: screen;
}

.overlay{
  background-blend-mode: overlay;
}

.difference {
  background-blend-mode: difference;
}
```

{{ EmbedLiveSample('Stacked_gradients', 200, 200) }}

> ### Blending gradients

Reviewer: This example uses gradients that aren't yet introduced. We've only seen linear gradients thus far. To create a useful example, limit the example to features that are already discussed plus the feature that is being added by the current example.

> In addition to transparency, stacked gradients can also be blended with the {{cssxref("background-blend-mode", "background-blend-mode")}} CSS property.

Reviewer: ```suggestion
In addition to transparency, stacking multiple semi-transparent gradients and stacking gradients over raster background images, gradients can be used with other CSS effects. In this example, we use the {{cssxref("background-blend-mode")}} CSS property to create blended effects.
```
background-blend-mode isn't directly related to gradients. Trying to make this more relevant to the page (though it still has the "we having seen these types of gradients yet" issue.

```html hidden

> <div class="lighten-gradients"></div>

Reviewer: include an "original" with no background-blend mode or any effects.

{{ EmbedLiveSample('Multiple_repeating_radial_gradients', 250, 150) }}

### Repeating conic gradients

This example uses {{cssxref("gradient/repeating-conic-gradient", "repeating-conic-gradient()")}} to create a gradient that rotates repeatedly around a center point. In this case, the declared color stops are repeated four times.

```html hidden
<div class="repeating-conic"></div>
```

```css hidden
div {
  width: 120px;
  height: 120px;
}
```

```css
.repeating-conic {
  background: repeating-conic-gradient(
    #66ccff 0% 8.25%,
    #6633ff 8.25% 16.5%,
    #ff3399 16.5% 25%
  );
}
```

{{ EmbedLiveSample('Repeating_conic_gradients', 120, 120) }}

### Multiple repeating conic gradients

> Just like linear and radial repeating gradients, you can stack multiple conic gradients on top of each other, creating interesting effects by using different `at <position>` values so the conic gradients don't overlap at their centers and different `from <angle>` values so the repeating effects don't line up. This example overlaps three semi-transparent repeating radial gradients that each repeat their color schemes four times. You will need to ensure either that the colors of the gradients on the top of the stack are partially transparent or use the {{cssxref("background-blend-mode")}} CSS property.

Reviewer: ```suggestion
Just like linear and radial repeating gradients, you can stack multiple conic gradients on top of each other, creating interesting effects by using different `at <position>` values so the conic gradients don't overlap at their centers and different `from <angle>` values so the repeating effects don't line up. This example overlaps three semi-transparent repeating radial gradients that each repeat their color schemes four times. To make overlapping gradients visible, you need to ensure either that the colors of the gradients on the top of the stack are partially transparent or use the {{cssxref("background-blend-mode")}} CSS property.
```

{{ EmbedLiveSample('Multiple_repeating_radial_gradients', 250, 150) }}

### Repeating conic gradients

> This example uses {{cssxref("gradient/repeating-conic-gradient", "repeating-conic-gradient()")}} to create a gradient that rotates repeatedly around a center point. The colors get cycled over and over as the gradient repeats.

Reviewer: ```suggestion
This example uses {{cssxref("gradient/repeating-conic-gradient", "repeating-conic-gradient()")}} to create a gradient that rotates repeatedly around a center point. In this case, the declared color stops are repeated four times.
```

```html hidden
<div class="repeating-conic"></div>
```

```css hidden
div {
  width: 120px;
  height: 120px;
}
```

```css
.repeating-conic {
  background: repeating-conic-gradient(
    #66ccff 0% 8.25%,
    #6633ff 8.25% 16.5%,
    #ff3399 16.5% 25%
  );
}
```

{{ EmbedLiveSample('Repeating_conic_gradients', 120, 120) }}

### Multiple repeating conic gradients

> Just like linear and radial repeating gradients, you can stack multiple conic gradients on top of each other. You will need to ensure either that the colors of the gradients on the top of the stack are partially transparent or use the {{cssxref("background-blend-mode", "background-blend-mode")}} CSS property

Reviewer: ```suggestion
Just like linear and radial repeating gradients, you can stack multiple conic gradients on top of each other, creating interesting effects by using different `at <position>` values so the conic gradients don't overlap at their centers and different `from <angle>` values so the repeating effects don't line up. This example overlaps three semi-transparent repeating radial gradients that each repeat their color schemes four times. You will need to ensure either that the colors of the gradients on the top of the stack are partially transparent or use the {{cssxref("background-blend-mode")}} CSS property.
```
* the second parameter of the cssxref macro is only needed if the text is different.
* mentioning this cool effect is created by putting the center of each conic gradient in a different location.
